client/extensions/woocommerce/state/sites/orders/test/actions.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { fetchOrders } from '../actions';
+import { LOADING } from 'woocommerce/state/constants';
+import useNock from 'test/helpers/use-nock';
+import { useSandbox } from 'test/helpers/use-sinon';
+import {
+	WOOCOMMERCE_ORDERS_REQUEST,
+	WOOCOMMERCE_ORDERS_REQUEST_FAILURE,
+	WOOCOMMERCE_ORDERS_REQUEST_SUCCESS,
+} from 'woocommerce/state/action-types';
+
+// The assembler `createOrderObject` fills out these fields on the order object.
+const formattedOrder = {
+	date_created: null,
+	date_modified: null,
+	date_paid: null,
+	date_completed: null,
+	line_items: [],
+	prices_include_tax: false,
+	totalPriceText: '',
+};
+
+describe( 'actions', () => {
+	describe( '#fetchSettingsGeneral()', () => {
+		const siteId = '123';
+
+		useSandbox();
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.get( '/rest/v1.1/jetpack-blogs/123/rest-api/' )
+				.query( { path: '/wc/v2/orders&_method=get' } )
```

Comment:
This will need rebased and updated to use `v3` now that my API bump PR has been merged.